### PR TITLE
Operate Deploy Guide UI parity check and fixes

### PR DESCRIPTION
**Pull request by g-murray**
[AAP-20550](https://issues.redhat.com/browse/AAP-20550)
[UI Parity Checklist](https://docs.google.com/document/d/1cJGUJsAGUW8Czlj5d0ysFtqoT1YzM5MFG6-3kWZCPpo/edit)

Location: Downstream > Titles > aap-operator-installation

### CODE REVIEW

File: downstream/modules/platform/proc-aap-migration-backup.adoc [hunk 1]
```diff
@@ -1,6 +1,20 @@
 [id="aap-migration-backup"]
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-aap-migration-backup.adoc:1]: There's no title for this module

File: downstream/modules/platform/proc-aap-migration-backup.adoc [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
+
+* Secret key secret
+* Postgresql configuration
+* Role-based Access Control for the namespaces on the new OpenShift cluster
+* The new OpenShift cluster must be able to connect to the previous PostgreSQL database
+
+[NOTE]
+====
+Secret key information can be located in the inventory file created during the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] via the Red Hat Customer portal.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-aap-migration-backup.adoc:15]:
```suggestion
You can store the secret key information in the inventory file before the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] via the Red Hat Customer portal.
```

I think an active verb would work better here, but my suggestion mihght not be right

File: downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc [hunk 3]
```diff
@@ -35,7 +35,7 @@ extra_settings:
     value: '12'
 ----
 +
-. Click btn[Reload] and btn[Save]. Return to the Form view.
+. Click btn:[Reload] and btn:[Save]. Return to the Form view.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc:38]: Are we supposed to use *Form* here? I'm not sure.

**Review by ariordan-redhat — COMMENTED** [review]
A few suggestions

File: downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc (hunk 3 above)
Comment (g-murray) [reply downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc:38]: "Form view" is the name of the radio button you can select, the other view they are currently in is "YAML view"

Comment (ariordan-redhat) [reply downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc:38]: I  phrased that wrong: what I meant to ask was whether **Form** was supposed to be bold for a UI element - I wasn't questioning the word "form"

File: downstream/modules/platform/proc-aap-migration-backup.adoc [hunk 4]
```diff
@@ -1,19 +1,35 @@
...
+====
+You can store the secret key information in the inventory file before the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] through the Red Hat Customer portal.
+====
+
 Before migrating your data from {PlatformNameShort} 2.x or earlier, you must back up your data for loss prevention. To backup your data, do the following:
 
 .Procedure
 . Log in to your current deployment project.
-. Run setup.sh to create a backup of your current data/deployment:
+. Run setup.sh to create a backup of your current data or deployment:
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-aap-migration-backup.adoc:24]:
```suggestion
. Run `setup.sh` to create a backup of your current data or deployment:
```

File: downstream/modules/platform/proc-aap-migration-backup.adoc [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
+
+* Secret key secret
+* Postgresql configuration
+* Role-based Access Control for the namespaces on the new OpenShift cluster
+* The new OpenShift cluster must be able to connect to the previous PostgreSQL database
+
+[NOTE]
+====
+You can store the secret key information in the inventory file before the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] through the Red Hat Customer portal.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-aap-migration-backup.adoc:17]:
```suggestion
You can store the secret key information in the inventory file before the initial {PlatformName} installation.
If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] through the Red Hat Customer portal.
```

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**